apps/modernization-ui/src/apps/search/layout/result/header/options/table/search-results-table-options.module.scss

```diff
@@ -10,3 +10,13 @@
     width: 1.75rem;
     height: 1.75rem;
 }
+
+.tooltip {
+    &[data-tooltip-position][aria-label] {
+        position: relative;
+        &::before {
+            content: attr(aria-label) !important;
+            font-size: 14px !important;
+        }
+    }
```

Comment:
question:  Why does this tool-tip have a different size than the others?

Reply:
in the figma for the tooltip I saw 14 px size so I added the styles just for this size which was different separate from the other styles, I thought It would not impact all of the tooltips but I can add the styles there as well if 14 px is okay?